scripts/release_scripts/deploy.py

```diff
@@ -398,6 +397,22 @@ def check_travis_and_circleci_tests(current_branch_name):
             'Please fix the circleci tests before deploying.')
 
 
+def create_release_doc():
+    """Asks the co-ordinator to create a doc for the current release."""
+    common.open_new_tab_in_browser_if_possible(
+        release_constants.RELEASE_DRIVE_URL)
+    common.open_new_tab_in_browser_if_possible(
+        release_constants.RELEASE_NOTES_TEMPLATE_URL)
+    common.open_new_tab_in_browser_if_possible(
+        release_constants.RELEASE_NOTES_EXAMPLE_URL)
+    common.ask_user_to_confirm(
+        'Please create a dedicated section for this release in the '
+        'release tracking document created by the QA Lead.\n'
+        'The three tabs in your browser point to: '
```

Comment:
Yes, I think it's ok since even if tabs cannot be opened automatically, the function `open_new_tab_in_browser_if_possible` asks user to open tabs before proceeding further.